exporter/splunkhecexporter/logdata_to_splunk.go

```diff
@@ -117,3 +118,51 @@ func nanoTimestampToEpochMilliseconds(ts pcommon.Timestamp) *float64 {
 	val := duration.Round(time.Millisecond).Seconds()
 	return &val
 }
+
+func mergeValue(dst map[string]any, k string, v any) int {
```

Comment:
Please don't return a value to be used solely in tests. Either use it in production code (debug log?) or don't return it

Reply:
Right! I was initially thinking of logging the number of dropped events, but now I am thinking of converting it to a string value instead of dropping it.